ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -85,6 +91,18 @@ module.exports = Marionette.LayoutView.extend({
             model: this.model
         }));
     },
+    showResult: function () {
```

Comment:
This might be taken care of in a related PR, but the references to permissions and title can be removed and replaced with just `model: this.model` since the variables already exist on the model object in the `QueryResult` component. 

Just need to update the references in `QueryResult` to pull them off the model correctly.

Example:
`this.options.title` would be replaced with `this.model.get('title')`. 
`this.options.permissions.accessIndividuals` would be replaced with `this.model.get('accessIndividuals')` ,etc